src/Splat/ServiceLocation/ModernDependencyResolver.cs

```diff
@@ -49,6 +49,12 @@ protected ModernDependencyResolver(Dictionary<Tuple<Type, string>, List<Func<obj
             _callbackRegistry = new Dictionary<Tuple<Type, string>, List<Action<IDisposable>>>();
         }
 
+        /// <inheritdoc />
+        public bool HasRegistration(Type serviceType)
+        {
+            return _registry.Any(x => x.Key.Item1 == serviceType);
```

Comment:
You might want to change this to use the Dictionary to check. I don't think Any() is dictionary aware. 

Eg a simple `return _registry.TryGetValue(serviceType, _);` should get similar results with O(1) results.

Reply:
cheers. will tweak later. about to pop out

Reply:
adjusting to

`
        public bool HasRegistration(Type serviceType)
        {
            var pair = Tuple.Create(serviceType, string.Empty);
            return _registry.ContainsKey(pair);
        }
`

Reply:
now using ValueTuple